src/SprykerEco/Zed/Adyen/Business/Hook/Mapper/MakePayment/KlarnaInvoiceMapper.php

```diff
@@ -17,13 +17,17 @@
 class KlarnaInvoiceMapper extends AbstractMapper
 {
     protected const REQUEST_TYPE = 'klarna';
+    protected const KLARNA_TAX_RATE_MULTIPLIER = 100;
 
     /**
      * @return string
      */
     protected function getReturnUrl(): string
     {
-        return '';
+        /** @var \SprykerEco\Zed\Adyen\AdyenConfig $config */
+        $config = $this->config;
```

Comment:
We might can do this by `@method` annotation in this class

Reply:
Added property annotation